_maps/_basemap.dm

```diff
@@ -4,15 +4,14 @@
 
 #ifndef LOWMEMORYMODE
 	#ifdef ALL_MAPS
-		#include "map_files/Pahrump-Sunset/Dungeons.dmm"
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset-Lower.dmm"
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset.dmm"
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset-Upper.dmm"
-		#include "map_files/Pahrump-Sunset/Pahrump-Sunset-Upper-2.dmm"
-		#include "map_files/Pahrump-Sunset/RockSprings.dmm"
-		#include "map_files/Pahrump-Sunset/RockSprings-Upper.dmm"
-		#include "map_files/Pahrump-Sunset/RedRiver.dmm"
-		#include "map_files/Pahrump-Sunset/Garland-City.dmm"
+		#include "_maps/map_files/coyote-bayou/Dungeons.dmm"
```

Comment:
You renamed the folder `coyote_bayou`.

Reply:
![1691890630255839](https://github.com/ARF-SS13/coyote-bayou/assets/13924177/4c7e643e-8be0-4e48-b5bf-30cd8d11b62d)